.conan/build.py

```diff
@@ -43,7 +43,7 @@ def upload_only_when_stable(self):
     def stable_branch_pattern(self):
         """ Only upload the package the branch name is like a tag
         """
-        return os.getenv("CONAN_STABLE_BRANCH_PATTERN", r"v?\d+\.\d+\.\d+-.*")
+        return os.getenv("CONAN_STABLE_BRANCH_PATTERN", r"v?\d+\.\d+\.\d+.*")
```

Comment:
If I am reading this properly, it would still pick up e.g. `v2.5.5-RC1`, which is not really a stable branch.

Also, you can rely on the "v" prefix always being there (generally I think the regex should be as strict as possible, to avoid publishing wrong things).

Reply:
sure! updating it now

Reply:
done!